xarray/tests/test_backends.py

```diff
@@ -2198,6 +2197,39 @@ class TestPyNioAutocloseTrue(TestPyNio):
     autoclose = True
 
 
+@requires_scipy
+@requires_pnc
+class TestPnc(NetCDF3Only, TestCase):
```

Comment:
`NetCDF3Only` class does not do anything (see line 127) but it is just a mixin that is used if the backend does not support int64 (see line 190).
Can you make this inherit `CFEncodedDataTest`?
 (if PNC only supports netcdf3, we need both `CFEncodedDataTest` and `NetCDF3Only` as `ScipyFileObjectTest` does)
It would cover more cases.

If pnc does not support a certain feature that `CFEncodedDataTest` tries to test, I think it would be OK now to explicitly skip it.

Reply:
Can you be more clear about this?

Reply:
You can see here
https://github.com/pydata/xarray/blob/7c2c43ce6b1792e1193635ab9b64fd248266f632/xarray/tests/test_backends.py#L127-L128

Inheriting `NetCDF3Only` does not do anything.

We have prepared general test cases `CFEncodedDataTest` and almost all of our backends are tested using this class.
I think pnc should be tested using `CFEncodedDataTest`.